<?xml version="1.0" encoding="ISO-8859-1"?>
<testsuite name="test_failure_output" errors="1" failures="32" tests="33" time="0.639" hostname="unknown" timestamp="2015-11-03T11:38:16">
  <properties/>
  <testcase classname="test_failure_output" name="test_empty">
    <failure><![CDATA[Expected empty expression was actually: 3.
In test_failure_output.m at line 12]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_not_empty">
    <failure><![CDATA[Expected non-empty expression was actually empty.
In test_failure_output.m at line 17]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_false">
    <failure><![CDATA[Expected false was actually true.
In test_failure_output.m at line 22]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_true">
    <failure><![CDATA[Expected true was actually false.
In test_failure_output.m at line 27]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_fail">
    <failure><![CDATA[This some custom fail message
In test_failure_output.m at line 31]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals">
    <failure><![CDATA[Data not equal:
  Expected : 3
  Actual   : 4
In test_failure_output.m at line 36]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_double_empty">
    <failure><![CDATA[Data not equal:
  Expected : []
  Actual   : 3
In test_failure_output.m at line 40]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_array">
    <failure><![CDATA[Data not equal:
  Expected : [1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16]
  Actual   : [1 2 3 4 5 6 7 8 9 10.000001 11 12 13 14 15]
In test_failure_output.m at line 46]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_arraymatrix">
    <failure><![CDATA[Data not equal:
  Expected : [1 2 3 4;5 6 7 8;9 10 11 12;13 14 15 16]
  Actual   : [1 2 3 4;5 6 7 8;9 10.000001 11 12;13 14 15 16]
In test_failure_output.m at line 54]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_string">
    <failure><![CDATA[Data not equal:
  Expected : 'foo'
  Actual   : 'fOO'
  Changes  :   ^^
In test_failure_output.m at line 59]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_scalar_cellstring">
    <failure><![CDATA[Data not equal:
  Expected : {'foo'}
  Actual   : {'fOO'}
  Changes  :    ^^
In test_failure_output.m at line 64]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_multiline_string">
    <failure><![CDATA[Data not equal:
  Expected : ['foo';'bar']
  Actual   : ['fuh';'bah']
  Changes  :    ^^     ^
In test_failure_output.m at line 70]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_multiline_string_asymmetric">
    <failure><![CDATA[Data not equal:
  Expected : ['foo·';'bar·']
  Actual   : ['fool';'barz']
  Changes  :      ^      ^
In test_failure_output.m at line 76]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_multiline_string_vs_singleline">
    <failure><![CDATA[Data not equal:
  Expected : ['foo';'bar']
  Actual   : ·'fool;·bar'·
  Changes  : ^    ^ ^    ^
In test_failure_output.m at line 82]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_cell">
    <failure><![CDATA[Data not equal:
  Expected : {3, 'foo'}
  Actual   : {3, 'fOo'}
In test_failure_output.m at line 89]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_struct">
    <failure><![CDATA[Data not equal at position .foo:
  Expected : {'hi', 'ho'}
  Actual   : {'ho', 'hi'}
  Changes  :    ^     ^
In test_failure_output.m at line 97]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_string_struct">
    <failure><![CDATA[Data not equal at position .foo:
  Expected : 'you say \to-'mah-(,)to'
  Actual   : 'you say \to-'mey-(,)to'
  Changes  :                ^^
In test_failure_output.m at line 107]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_scalar_cellstring_struct">
    <failure><![CDATA[Data not equal at position .foo:
  Expected : {'you say \to-'mah-(,)to'}
  Actual   : {'you say \to-'mey-(,)to'}
  Changes  :                 ^^
In test_failure_output.m at line 117]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_nested_struct">
    <failure><![CDATA[Data not equal at position .tee.foo:
  Expected : {'hi', 'ho'}
  Actual   : {'ho', 'hi'}
  Changes  :    ^     ^
In test_failure_output.m at line 126]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_nested_struct_multiple_differences">
    <failure><![CDATA[Data not equal at position .tee.bar:
  Expected : [3 4]
  Actual   : [3 5]
Data not equal at position .tee.foo:
  Expected : {'hi', 'ho'}
  Actual   : {'ho', 'hi'}
  Changes  :    ^     ^
In test_failure_output.m at line 136]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_nested_struct_many_differences">
    <failure><![CDATA[Data not equal at position .bar:
  Expected : [3 4]
  Actual   : [3 5]
Data not equal at position .foo:
  Expected : {'hi', 'ho'}
  Actual   : 42
Data not equal at position .tee.bar:
  Expected : [3 4]
  Actual   : [3 5]
... more differences detected, but not displayed.
In test_failure_output.m at line 148]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_structarray">
    <failure><![CDATA[Data not equal:
  Expected : [{foo:{'hi', 'ho'}; bar:3}, {foo:{'hi', 'ho'}; bar:4}]
  Actual   : [{foo:{'hi', 'ho'}; bar:3}, {foo:{'ho', 'hi'}; bar:4}]
In test_failure_output.m at line 156]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_object">
    <failure><![CDATA[Data not equal:
  Expected : Time Series Object: unnamed

Time vector characteristics

      Length                12
      Start time            0 seconds
      End time              11 seconds

Data characteristics

      Interpolation method  linear
      Size                  [1   1  12]
      Data type             double
  Actual   : Time Series Object: unnamed

Time vector characteristics

      Length                12
      Start time            0 seconds
      End time              11 seconds

Data characteristics

      Interpolation method  linear
      Size                  [1   1  12]
      Data type             double
In test_failure_output.m at line 166]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_handle">
    <failure><![CDATA[Data not equal:
  Expected : 0
  Actual   : 1
In test_failure_output.m at line 181]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_functionhandle">
    <failure><![CDATA[Data not equal:
  Expected : @(x)x*x+x
  Actual   : @(x)x+x*x
In test_failure_output.m at line 197]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_equals_javaobject">
    <failure><![CDATA[Data not equal:
  Expected : hiho
  Actual   : hiHo
In test_failure_output.m at line 203]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_not_equals">
    <failure><![CDATA[Expected and actual are equal.
In test_failure_output.m at line 208]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_error">
    <failure><![CDATA[Should be error, but 3 alone isn't.
Error expected, but none occurred.
In test_failure_output.m at line 213]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_typed_error">
    <failure><![CDATA[Error expected, but none occurred.
In test_failure_output.m at line 218]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_wrong_typed_error">
    <failure><![CDATA[Error occurred, but did not match criteria. Expected error identifier 'MATLAB:SomeError' actually was 'MATLAB:m_improper_grouping'.
In test_failure_output.m at line 222]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_warning">
    <failure><![CDATA[No warning MATLAB:rmpath:DirNotFound when executing function @()3.
In test_failure_output.m at line 228]]></failure>
  </testcase>
  <testcase classname="test_failure_output" name="test_implementation_fault">
    <error><![CDATA[Undefined function or variable 'unknown_function_call'.
In test_failure_output.m at line 232
In run_test.m at line 21
In run_test.m at line 50
In run_suite.m at line 86
In run_suite_collection.m at line 126
In run_suite_collection.m at line 45
In recursive_test_run.m at line 27]]></error>
  </testcase>
  <testcase classname="test_failure_output" name="test_nested_functions">
    <failure><![CDATA[Nested function test.
In test_failure_output.m at line 240
In test_failure_output.m at line 236]]></failure>
  </testcase>
  <system-out/>
  <system-err/>
</testsuite>
